CHANGELOG.md

```diff
@@ -4,6 +4,58 @@ All notable changes to this project will be documented in this file.
+
+### Added
+- [End to end RAG examples](./RAG/examples/) enhancements
+  - [Single command deployment](./README.md#try-it-now) for all the examples using docker compose.
+  - All end to end RAG examples are now more encapsulated with documentation, code and deployment assets residing in dedicated example specific directory.
+  - Segregated examples into [basic and advanced RAG](./RAG/examples/) with dedicated READMEs.
+  - Added reranker model support to [multi-turn RAG example](./RAG/examples/advanced_rag/multi_turn_rag/)
+  - Removed python dev packages from containers to enhance security
+  - Updated to latest version of [langchain-nvidia-ai-endpoints](https://python.langchain.com/v0.2/docs/integrations/providers/nvidia/)
+- [Speech support using rag-playground]((./docs/riva-asr-tts.md))
+  - Added support to access [RIVA speech models from NVIDIA API Catalog](https://build.nvidia.com/explore/speech).
+  - Speech support in UI is opt-in.
+- Documentation enhancements
+  - Added more comprehensive [how-to guides](./README.md#how-to-guides) for end to end RAG examples.
+  - Added [example specific architecture diagrams](./RAG/examples/basic_rag/langchain/) in each example directory.
+- Added a new industry specific [top level directory](./industries/)
+  - Added [health care domain specific Medical Device Training Assistant RAG](./industries/healthcare/medical-device-training-assistant/).
+- Added notebooks showcasing new usecases
+  - [Basic langchain based RAG pipeline](./RAG/notebooks/langchain/langchain_basic_RAG.ipynb) using latest Nvidia API Catalog connectors.
+  - [Basic llamaindex based RAG pipeline](./RAG/notebooks/llamaindex/llamaindex_basic_RAG.ipynb) using latest Nvidia API Catalog connectors.
+  - [NeMo Guardrails with basic langchain RAG](./RAG/notebooks/langchain/NeMo_Guardrails_with_LangChain_RAG/).
+  - [NVIDIA NIMs using NeMo Guardrails based RAG](./RAG/notebooks/langchain/Using_NVIDIA_NIMs_with_NeMo_Guardrails/).
+  - [Using NeMo Evaluator using Llama 3.1 8B Instruct](./RAG/notebooks/nemo/Nemo%20Evaluator%20Llama%203.1%20Workbook/).
+  - [Agentic RAG pipeline with Nemo Retriever and NIM for LLMs](./RAG/notebooks/langchain/agentic_rag_with_nemo_retriever_nim.ipynb).
+- Added new `community` (before `experimental`) example
+  - Create a simple UI to interact with different [selectable NIM endpoints](./community/llm-prompt-design-helper/). The provided interface of this project supports designing a system prompt to call the LLM.
+
+### Changed
+- Major restructuring and reorganisation of the assets within the repository
+  - Top level `experimental` directory has been renamed as `community`.
+  - Top level `RetrievalAugmentedGeneration` directory has been renamed as just `RAG`.
+  - The docker compose files inside top level `deploy` directory has been migrated to example specific directories under `RAG/examples`. The vector db and on-prem NIMs deployment files are under `RAG/examples/local_deploy`.
```

Comment:
Docker Compose, example-specific, vector database, s/NIMs/NIM microservices/

Reply:
Thanks made all the changes